### DISCUSSION TITLE
Incompatibility with frames.js

### DISCUSSION THREAD

**Discussion by dotSaeid** [zhensherlock/watermark-js-plus#522] [General]
I want to use this library in frames.js, which I also made with nextjs, but it is not compatible!
It has the following problem!
The document is not defined
Is there a solution? please guide me

**Comment by zhensherlock** [comment]
https://stackblitz.com/edit/stackblitz-starters-pv2mvx?file=app%2Fwatermark.ts
Hi,This is an example with nextjs.
